lib/scidata/squad.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Scidata.Squad do
+  @moduledoc """
+  Module for downloading the [Squad dataset](https://rajpurkar.github.io/SQuAD-explorer).
+  """
+
+  require Scidata.Utils
+  alias Scidata.Utils
+
+  @base_url "https://rajpurkar.github.io/SQuAD-explorer/dataset/"
+  @dataset_file "train-v1.1.json"
```

Comment:
We can also download the test file, `dev-v1.1.json`, and make it available with a `download_test/0` function :)

Reply:
Good idea, i will do that!